metadata.txt

```diff
@@ -12,8 +12,8 @@ email=matteo.ghetta@faunalia.eu
 
```

Comment:
Hey @ghtmtt can you change the `name` field on line `6` to `QGreenland Custom`? We decided that we are going to brand the plugin as `QGreenland Custom` and the zip package as `QGreenland Core`.